helpers.sh

```diff
@@ -61,6 +61,10 @@ btcpay_expand_variables() {
     if [[ "$BTCPAY_HOST" != *.local ]] && [[ "$BTCPAY_HOST" != *.lan ]]; then
         BTCPAY_ANNOUNCEABLE_HOST="$BTCPAY_HOST"
     fi
+    
+    if [ ! -z "$BTCPAY_ADDITIONAL_HOSTS" ]; then 
+        BTCPAY_ADDITIONAL_HOSTS=",${BTCPAY_ADDITIONAL_HOSTS// }"
```

Comment:
what is `// }`

Reply:
It removes any ` ` in the var
eg 
```
TEST="hello1world"
TEST="${TEST//1}"
echo "$TEST" //helloworld
```

Reply:
Can you replace any `;` by `,` so that people don't complain if they messed up?

Reply:
@Kukks you know what? I think I will just change the nginx template to separate on both `,` and `;` as well as ignoring empty entries so that you don't have to do this hack.

Reply:
Sure, should I remove from this pull request then?

Reply:
no, just the hack part here.

Reply:
@Kukks  can you remove this part and use 0.7.6 for docker-gen ?
Also change `,` to `;` in the nginx template.